differs/rpm_diff.go

```diff
@@ -123,6 +123,20 @@ func rpmDataFromContainer(image pkgutil.Image) (map[string]util.PackageInfo, err
 
 		defer client.RemoveImage(imageName)
 		defer logrus.Infof("Removing image %s", imageName)
+	} else if image.IsCloud() {
+		// if it's a remote image and not in the daemon, we need to pull it
+		if _, err := client.InspectImage(imageName); err != nil {
```

Comment:
if `err == nil` here, we should probably log and exit yeah? otherwise we have an invalid image and this will fail somewhere else down the line

Reply:
Could you elaborate on why the image would be invalid in such case? I `err == nil` then I would expect the tool to use the already present image. I agree that it should be logged.

Reply:
ping @nkubala

Reply:
Ah sorry, I confused myself with the error cases here. Disregard my original comment.

I think it might make sense to specify an option to users (through a flag or something) to always pull, but I don't think we want to do that by default. One of the major use cases is comparing locally built images with their remote counterparts, and I don't think we should assume that users will always use image tags appropriately :) either way this is outside the scope of this PR imo, but feel free to file a separate issue to track this